base/src/com/google/idea/blaze/base/toolwindow/ConsoleView.java

```diff
@@ -237,19 +237,17 @@ void println(PrintOutput output) {
   }
 
   private void println(String text, OutputType outputType) {
-    if (outputType == OutputType.PROCESS) {
-      text = text.stripTrailing();
-    }
-
     ansiEscapeDecoder.escapeText(
         text,
         outputType == OutputType.ERROR ? ProcessOutputTypes.STDERR : ProcessOutputTypes.STDOUT,
         (t, k) -> consoleView.print(t, ConsoleViewContentType.getConsoleViewType(k)));
-    consoleView.print(
-        "\n",
-        outputType == OutputType.ERROR
-            ? ConsoleViewContentType.ERROR_OUTPUT
-            : ConsoleViewContentType.NORMAL_OUTPUT);
+    if (outputType != OutputType.PROCESS) {
```

Comment:
Using separators "as is" for the process and adding potentially omitted ones for other output types.